packages/cli-hooks/src/get-hooks.js

```diff
@@ -43,11 +43,13 @@ if (fs.realpathSync(process.argv[1]) === fileURLToPath(import.meta.url)) {
  * @returns {SDKInterface} Information about the hooks currently supported.
  */
 export default function getHooks() {
+  // NODE_NO_WARNINGS=1 silences node process warnings. These warnings can occur
+  // on some node version (e.g. v23.2.0) and cause invalid JSON hook responses.
   return {
     hooks: {
-      doctor: 'npx -q --no-install -p @slack/cli-hooks slack-cli-doctor',
-      'check-update': 'npx -q --no-install -p @slack/cli-hooks slack-cli-check-update',
-      'get-manifest': 'npx -q --no-install -p @slack/cli-hooks slack-cli-get-manifest',
+      doctor: 'NODE_NO_WARNINGS=1 npx -q --no-install -p @slack/cli-hooks slack-cli-doctor',
+      'check-update': 'NODE_NO_WARNINGS=1 npx -q --no-install -p @slack/cli-hooks slack-cli-check-update',
+      'get-manifest': 'NODE_NO_WARNINGS=1 npx -q --no-install -p @slack/cli-hooks slack-cli-get-manifest',
```

Comment:
🙏 Nice find! I hadn't noticed these warnings in the versions we officially support - `18`, `20`, `22` for now - but think this is a nice and quick guard for those on the cutting edge.

Reply:
The Node [release schedule](https://nodejs.org/en/about/previous-releases) is so interesting to me!

I believe odd versions are treated as a "current" release and act as an upgrading version to the next even version. These odd versions never get long term support 😅

We might want to discuss testing with the latest current version across the SDKs to make sure bumping to the next current LTS goes smooth.

Supporting that version IMO is another question 🤔